src/models/data.py

```diff
@@ -234,9 +263,11 @@ def ds_folder_to_np(self,
         y_np = y_np.reshape(y_np.shape[0], y_np.shape[1], y_np.shape[2] * y_np.shape[3])
         y_np = np.moveaxis(y_np, -1, 0).reshape(-1, 1)
 
-        if self.normalizing_array is not None:
+        if (self.normalizing_array is not None) and (self.normalizing_array_ym is not None):
             x_np = (x_np - self.normalizing_array['mean']) / (self.normalizing_array['std'])
             train_latlons = train_latlons / [90, 180]
+            yearly_mean_np = (yearly_mean_np - self.normalizing_array_ym['mean']) / \
```

Comment:
so we're not including the yearly mean data we are also normalizing the data by the annual mean?

Does that make sense to duplicate that information? Wouldn't normalisation by the global mean and then including the annual mean have all of that information already incorporated?

Reply:
`normalizing_array_ym` is the global mean information, fitted to an array that has the same dimensions as the yearly mean.

So normalization only happens using the global mean